files/ru/learn/css/building_blocks/the_box_model/index.html

```diff
@@ -282,91 +284,75 @@ <h3 id="Padding">Padding</h3>
 
 <p><strong>Если вы измените значения padding в классе <code>.box</code> в примере ниже, вы можете увидеть что это изменяет положение текста относительно поля.</strong></p>
```

Comment:
```suggestion
<p><strong>Если вы измените значения padding для класса <code>.box</code> в примере ниже, то увидите, что это изменяет положение текста внутри элемента.</strong></p>
```
В оригинале там:
`where the text begins in relation to the box.`